arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -1354,8 +1353,7 @@
     },
     "SharedAccessAuthorizationRuleCreateOrUpdateParameters": {
       "properties": {
-        "properties": {
-          "x-ms-client-flatten": true,
```

Comment:
are you sure you want to remove x-ms-flatten here?  You will end up with code like this: Properties.Properties